src/ucp/dt/dt.c

```diff
@@ -142,3 +142,38 @@ size_t ucp_dt_pack(ucp_worker_h worker, ucp_datatype_t datatype,
     state->offset += result_len;
     return result_len;
 }
+
+ucs_status_t ucp_dt_query(ucp_datatype_t datatype, const void *buffer, size_t count, ucp_datatype_attr_t *attr)
```

Comment:
pls check code format
https://github.com/openucx/ucx/wiki/Code-style-checking

Reply:
Moved `count` and `attr` parameters to next line. (Unfortunately, git-clang-format from the UCX LLVM fork fails to work properly on my local system.)